packages/fiori/src/ShellBar.ts

```diff
@@ -588,6 +588,7 @@ class ShellBar extends UI5Element {
 		this._coPilotIcon = !this._coPilotPressed ? ShellBar.CO_PILOT_ICON_PRESSED : ShellBar.CO_PILOT_ICON_UNPRESSED;
 		button.icon = this._coPilotIcon;
 		this._coPilotPressed = !this._coPilotPressed;
+		button.classList.toggle("ui5-shellbar-co-pilot-pressed", this._coPilotPressed);
```

Comment:
If possible, it's recommended to use the template and add/remove the class based on state. 
You can reuse the `_coPilotPressed`. We have these class maps, there are used in many components, 

get classes():
   coPilotBtn: {
       "ui5-shellbar-co-pilot-pressed": this._coPilotPressed
   }
}

and then in the template you can have {{classes.coPilotBtn}}